unabridged-works-volume-4/original/02-life-and-immortality.adoc

```diff
@@ -3200,10 +3413,13 @@ Amen.
 [.inline]
 ==== XX. Concerning Baptism
 
-"`He that believeth, and is baptised, shall be saved;
-but he that believeth not, shall be damned.`" Mark 16:16.
+[quote.epigraph, , Mark 16:16]
```

Comment:
so, this should be a `quote.section-epigraph` not a `quote.epigraph` -- regular epigraphs can only come at the very beginning of the first file in a document. I'm not sure if I ever told you that.

That said, I slightly lean towards making this just a `[quote.scripture]` instead of a section-epigraph.  The reason is, I think of an epigraph as kind of being an ornamental verse that helps establish the _theme_ of a section or book. So it's not really a part of the _content_. But here he immediately starts talking about this verse, so that's why I lean towards just making it a regular scripture block quote, even though it's at the very beginning of a section. Hopefully that makes sense.

Reply:
Yes, that makes sense!